music_server.py

```diff
@@ -1,10 +1,33 @@
-from flask import Flask
+from flask import Flask, jsonify, request
+from album_search import *
+import os
 from waitress import serve
-
 import logging
+from album_search import *
 
 app = Flask(__name__)
 
+#SEARCH_FOLDER_PATH = r"C:\Users\davis_g7\OneDrive\Documents\Duq-CPMA-536-Spring-2024\Music"
+SEARCH_FOLDER_PATH = r"C:\insert\path\to\your\music\directory\here"
```

Comment:
Instead of using absolute paths, we should use relative paths.

This can be changed to the following:

```python
SEARCH_FOLDER_PATH = r"Music"
```